scripts/rebuild_languages.py

```diff
@@ -33,104 +35,120 @@ def downloadSheet(url):
     return list(reader)
 
 
-languageSpreadsheetURL = (
-    "https://docs.google.com/"
-    "spreadsheets/d/1woTU8dZCHJh7yvdk-N1kgQBUj4Sn3SdRsbKgn6ltJQs/"
-)
+def main(wip):
```

Comment:
Please rename this variable to `includeWorkInProgress`.

Reply:
Maybe rename the command line flag to `--include-wip` to make this more explicit.

Reply:
done.